backend/src/utils/security.py

```
@@ -1,13 +1,41 @@
+    """
+    Create a Fernet instance from APP_SECRET_KEY.
+
+    APP_SECRET_KEY must be a **Fernet key** (urlsafe base64-encoded 32 bytes).
+    """
+
+    key = (APP_SECRET_KEY or "").strip().strip('"').strip("'")
+    if not key:
+        raise ValueError(
+            "APP_SECRET_KEY is not set. It must be a Fernet key (e.g. generated via "
+            "`python -c \"from cryptography.fernet import Fernet; print(Fernet.generate_key().decode())\"`)."
+        )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)